Dockerfile

```diff
@@ -0,0 +1,125 @@
+ARG REACT_APP_NAME=frontend
```

Comment:
You'd be better of splitting this into two dockerfiles and putting them into their respective directories (frontend and backend)